<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_form</class>
 <widget class="QWidget" name="login_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/arts.png</normaloff>:/icons/arts.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font-family: url(:/fonts/OpenSans-Regular.ttf);
	font-size: 35px;
}
#login_form {
	background-image: url(:/images/pic1.jpg);
	background-repeat: no-repeat;
}

QFrame#login_frame {
	background-color: rgba(238, 238, 238, .8);
}
QLabel#login_header {
	color: #3e2723;
}

QToolButton#login_logo {
    background-color: #ec407a;
	border-radius: 45px;
}

QLineEdit {
	background-color: #fff;
	color: #ec407a;
	border: 1px soild;
	padding: 0 10px;
	font-size: 18px;
}

QLabel#user_pic, QLabel#pass_pic  {
	background-color:#ec407a;
}

QPushButton#login_button {
	background-color: #ec407a;
	border: 1px soild;
	color: #fff;
	font-size: 18px;
	font-weight: bold;
}

QPushButton#login_button:hover {
	background-color: #ad1457;
	border: 1px soild;
	color: #fff;
}

QLabel#error_message {
	 color: #d50000;
	font-size: 14px;
}</string>
  </property>
  <widget class="QFrame" name="login_frame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>400</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="login_header">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Login Here</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>195</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>290</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>35</height>
     </size>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Username ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_pic">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources/res.qrc">:/icons/user24-white.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pass_pic">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>195</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources/res.qrc">:/icons/pass-24-white.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="error_message">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>:/fonts/OpenSans-Regular.ttf</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="login_logo">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>61</y>
     <width>91</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources/res.qrc">
     <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>login_button</tabstop>
  <tabstop>login_logo</tabstop>
 </tabstops>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
